index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio Hana Sabih</title>
+    <link rel="stylesheet" href="style.css" />
+    <script src="https://code.iconify.design/iconify-icon/1.0.2/iconify-icon.min.js"></script>
+  </head>
+  <body>
+    <header>
+      <a href=""><h5>Welcome</h5></a>
+      <nav><img src="./images/icons/Union.png" alt="Union" /></nav>
+    </header>
+    <section class="headline">
+      <h1>
+        Hey there. I'm Mariane<br />
+        <span>I'm a software developer</span>
+      </h1>
+      <!-- <h3>Software Developer</h3> -->
+
+      <p>
+        I can help you build a product , feature or website Look through some of
+        my work and experience! If you like what you see and have a project you
+        need coded, don't hestiate to contact me.
+      </p>
+      <div class="icons-contact">
+        <img src="./images/icons/twitter.png" alt="twitter" />
+        <img src="./images/icons/Linkedin.png" alt="Linkedin" />
+        <img src="./images/icons/medium.png" alt="medium" />
+        <img src="./images/icons/github.png" alt="github" />
+        <img src="./images/icons/angellist.png" alt="angellist" />
```

Comment:
- [ ] Good job 👇  implementing the suggested changes about the icons 👍 
![image](https://user-images.githubusercontent.com/78251927/205432075-32f91153-c01c-47a3-ac90-74ec21343725.png)

- [ ] **[OPTIONAL]** Since the previous reviewer did not point out this [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md), it is marked **OPTIONAL** but highly recommended,
- [ ] The list of `icons` should use a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag). apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href.